src/assets/stylesheets/components/_form-button.scss

```diff
@@ -0,0 +1,8 @@
+.form-button {
+  width: 100%;
+  height: rem(50px);
+
+  border-radius: rem(10px);
+
+  font-size: rem(17px);
```

Comment:
@gutakk Should we set the `font-weight` here?
![Screen Shot 2564-06-18 at 19 22 10](https://user-images.githubusercontent.com/1772999/122560227-8e73da00-d06a-11eb-8e43-810c465291e8.png)

Reply:
Fixed in 8bb4e8ddce1e02e787bd57162bd62e08fae8c23e